### ISSUE TITLE
Poor rendering of headers?

### ISSUE THREAD

**Issue by larsks** [jaspervdj/patat#81]
I'm feeding patat the follow input:

```
# An example deck

## This is a test

This is my first slide.

Here is some information.

## This ought to be a slide title

The quick brown fox:

- Jumped over the lazy dog.
- Injured itself upon landing.
- This is unsafe workplace behavior.
```

While patat is obviously *recognizing* the headers, it's not rendering them very well: e.g., the rendered headers still include the `##` markers.  For example, the title slide renders like this:

![Screenshot from 2019-11-26 10-13-05](https://user-images.githubusercontent.com/82622/69646194-5f83b680-1035-11ea-958c-b112e7fb7e8c.png)

And the first slide renders like this:

![Screenshot from 2019-11-26 10-14-10](https://user-images.githubusercontent.com/82622/69646266-80e4a280-1035-11ea-8942-f39388d91927.png)

Is this expected behavior?

**Comment by jaspervdj** [comment]
Yes, this is expected, to stick somewhat close to markdown format (which is good if people seeing the presentation recognize it's a header IMO).  I didn't want to draw a line below it because that takes up a whole line of screen real estate.  Do you have other suggestions/preferences?  I would be happy to make the title rendering more configurable but I'm not sure about changing the default.

**Comment by larsks** [comment]
I agree re: not changing the default, but I would love to be able to drop the `#` markers from the headers and to use some sort of styling to indicate "header-ness".

If styling options included underlined/not underlined, that would be cool.

**Comment by halloleo** [comment]
I concur, an option to hide the # sign would be fantabulous!
